wing-redis/README.md

```diff
@@ -0,0 +1,53 @@
+# Wing Redis Client Library
+
+This library provides a Wing client for Redis.
+
+## Installation
+
+Use `npm` to install this library:
+
+```sh
+npm i @winglibs/wing-redis
```

Comment:
Please rename to just `redis`

Reply:
I have changed it manually everywhere possible. Should I change the `wing-redis.yaml` file manually at `.github/workflows`?